test/core/models/StanzaHelper.scala

```diff
@@ -36,7 +36,7 @@ trait StanzaHelper extends TestConstants {
     Phrase(Vector("Text 8", "Welsh: Text 8"))
   )
 
-  val exclusiveSequencePhrases: Vector[Phrase] = phrases ++ Vector(
+  val rawExclusiveSequencePhrases: Vector[Phrase] = phrases ++ Vector(
```

Comment:
If this has been refactored to use rawExclusiveSequencePhrases ... then does this require to be used at lines 108, 111 - 113 ?

Reply:
no, different usage

Reply:
Discussed